lab_1_classify_by_unigrams/main.py

```diff
@@ -105,6 +218,14 @@ def detect_language(
 
     In case of corrupt input arguments, None is returned
     """
+    if profiles_bad_input(unknown_profile) == None or profiles_bad_input(profile_1) == None or profiles_bad_input(profile_2) == None:
+        return None
+    mse_1 = compare_profiles(unknown_profile, profile_1)
+    mse_2 = compare_profiles(unknown_profile, profile_2)
+    if mse_1 <= mse_2:
+        return profile_1["name"]
+    else:
```

Comment:
there is no need in else. Can you answer why?

Reply:
oh, thank you. exactly, if we returned before, then the next return will not be implemented. otherwise, it will be.

done